gammapy/modeling/parameter.py

```diff
@@ -261,6 +270,14 @@ def max(self, val):
         else:
             self._max = self._set_quantity_str_float(val)
 
+    @property
+    def factor_max(self):
+        """Factor maximum as a float.
+
+        This ``factor_max = transform(max)`` is for the optimizer interface.
```

Comment:
For clarity, I would keep the previous definition and add the new one as most of the time transform is identity:
```
In general, when no transform is applied, ``factor_min = min / scale`` else ``factor_min = transform(min)``
```